lib/hunts/standard.py

```diff
@@ -154,6 +154,7 @@ class DeleteHuntFlow(flow.GRRFlow):
 
   @flow.StateHandler()
   def Start(self):
+    protect_results = config_lib.CONFIG["AdminUI.protect_hunt_results"]
```

Comment:
Similar to above I mostly did that to make the if X and Y line easier to read.  I can just use it directly if that is preferred